pulpcore/pulpcore/app/serializers/base.py

```diff
@@ -171,7 +171,7 @@ def _validate_relative_path(self, path):
         """
         # in order to use django's URLValidator we need to construct a full url
         base = "http://localhost"  # use a scheme/hostname we know are valid
-        validate = URLValidator()
+        validate = URLValidator(message=_("Relative path cannot contain spaces."))
```

Comment:
Suggestion:

Instead of doing this, you could move lines 177-178 up here, and raise a validation error with a specific error  message in the condition  ```if ' ' in path```

Reply:
Ok, good point!